disttask/loaddata/wrapper.go

```diff
@@ -84,6 +99,16 @@ func makeTableRegions(ctx context.Context, task *TaskMeta, concurrency int) ([]*
 	return mydump.MakeTableRegions(ctx, dataDivideConfig)
 }
 
+func rebaseRowID(rowIDBase int64, tableRegions []*mydump.TableRegion) {
```

Comment:
this function can be moved inside makeTableRegions? And I need to know what's the meaning of task/subtask to understand why we need rebase